packages/expo-dev-menu/android/src/main/java/expo/modules/devmenu/DevMenuUtils.kt

```diff
@@ -0,0 +1,34 @@
+package expo.modules.devmenu
+
+import android.app.Application
+import com.facebook.react.ReactNativeHost
+import com.facebook.react.ReactPackage
+import org.unimodules.core.interfaces.Package
+
+fun getExpoModules(application: Application): List<Package> {
+  val basePackageListClass = Class.forName(application.packageName + ".generated.BasePackageList")
+  val getPackageList = basePackageListClass.getMethod("getPackageList")
+  @Suppress("UNCHECKED_CAST")
+  return getPackageList.invoke(basePackageListClass.newInstance()) as List<Package>
+}
```

Comment:
```suggestion
fun Application.getExpoModules(): List<Package> {
  val basePackageListClass = Class.forName(this.packageName + ".generated.BasePackageList")
  val getPackageList = basePackageListClass.getMethod("getPackageList")
  @Suppress("UNCHECKED_CAST")
  return getPackageList.invoke(basePackageListClass.newInstance()) as List<Package>
}
```
What would say for converting it to the class extension?